### DISCUSSION TITLE
logo和favicon在chrome和edge无法正常显示

### DISCUSSION THREAD

**Discussion by 2w1nd** [ppoffice/hexo-theme-icarus#1027] [Q&A]
以下是我的配置文件
![image](https://user-images.githubusercontent.com/64266855/151682950-d7c6d739-e1ef-483f-b3db-77a7044ac470.png)
edge中：
![image](https://user-images.githubusercontent.com/64266855/151682971-bf276dbe-3883-4cf4-aced-c9ab52dbbcb3.png)
资源已正常请求
![image](https://user-images.githubusercontent.com/64266855/151682980-2ad531e7-c904-4905-ac3f-474fec0ac03a.png)
资源文件
![image](https://user-images.githubusercontent.com/64266855/151683051-1c5db229-1bac-4de7-8854-8e42aca53fa2.png)
chrome中是同样的，原来的默认logo和favicon可以正常显示，换了自己的就不行了
但在firfox中可以显示
![image](https://user-images.githubusercontent.com/64266855/151683007-47367625-4c59-4563-9f66-655c692dd024.png)
已排查由浏览器缓存导致的问题

**Comment by 2w1nd** [comment]
问题大致解决，我将svg换成了png，估计是svg图片在除firefox以外的浏览器的解析问题，我用ps导出的svg只能在firefox显示。。还期待有人是回答如何导出在任何平台显示正常的svg

**Reply by YiSunYiMao** [reply]
试一下将svg转为ico格式
